docs/modules/con-proxy-overview.adoc

```diff
@@ -11,9 +11,9 @@ In an ideal world, such functionality would be implemented directly in Apache Ka
 But there are numerous practical reasons that can prevent this, for example:
 
 * Organizations having very niche requirements which are unsuitable for implementation directly in Apache Kafka.
-* Functionality which requires changes to Kafka's public API and which the Apache Kafka project is unwilling to implement.
+* Functionality which requires changes to Kafka's public API and which the Apache Kafka&#174; project is unwilling to implement.
```

Comment:
Isn't the obligation to acknowledge the trademark at the first use within a document?  As this file is a fragment that gets included in a larger whole, I don't think we should apply it here.

Reply:
My take was first use was the minimum to comply and rather than trying to figure out the first usage I went for search and replace. Both options are equally valid

Reply:
Won't that read funny in the resulting output document?   It'll flip between attributed references and ones without without obvious rhyme nor reason.

Reply:
Why would it read funny if we have added the registered symbol to all usages? 

If you mean "Apache Kafka (r)" vs "Kafka" then I don't think adding the symbol adds or removes confusion or inconsistency.

I'm happy to drop this commit and just merge the path fix as I did the search and replace while watching the team call. It was only once I had pushed it up I found your PR.